README.md

```diff
@@ -355,6 +356,20 @@ To see a list of all available command line options, use the following command.
 freec --help
 ```
 
+#### `--from=LANG`, `--to=LANG`
+
+By default the compiler translates from Haskell to Coq.
+The `--from` and `--to` options select a different frontend and backend respectively.
+Currently the `haskell` frontend and the `coq` backend are available.
+
+For debugging purposes there also exists a `ir` frontend and backend
+that parses or prints the intermediate representation used by the compiler respectively.
```

Comment:
[Write one sentence per line](https://github.com/FreeProving/guidelines/blob/main/CONTRIBUTING.md#write-one-sentence-per-line) in Markdown files.

```suggestion
For debugging purposes there also exists a `ir` frontend and backend that parses or prints the intermediate representation used by the compiler respectively.
```